### PR TITLE
What's new 8.12

### LINKED ISSUE #4465: What's new in 8.12
Please add your features and enhancements for `8.12`. Don't forget to include the related PR link! 

### Rules Management 

- [x] JSON diff for Elastic prebuilt rule updates (https://github.com/elastic/security-docs/pull/4463).
When Elastic updates a prebuilt detection rule, you can examine the latest version before you update to it. The rule details flyout in **Rule Updates** displays a side-by-side JSON comparison of the rule’s **Base version** (what you currently have installed) and the **Update version** that you can choose to install. 

### Detection Engine 

- [x] Alert suppression for threshold rules (https://github.com/elastic/security-docs/pull/4467).
[Alert suppression](https://www.elastic.co/guide/en/security/master/alert-suppression.html) now supports the threshold detection rule type. You can use it to reduce the number of repeated or duplicate detection alerts created by a threshold rule.
- [x] Assign users to alerts (https://github.com/elastic/security-docs/pull/4476).
You can now [assign users to alerts](https://www.elastic.co/guide/en/security/master/alerts-ui-manage.html#assign-users-to-alerts) that you want them to investigate, and manage alert assignees throughout an alert’s lifecycle. Assigned alerts are filterable and you can find assignees by adding the `kibana.alert.workflow_assignee_ids` field to the Alerts table or by opening an alert's details.

### Explore

- [ ] Add features here.

### Investigations

- [x] New advanced setting for excluding cold and frozen tier data from analyzer queries (https://github.com/elastic/security-docs/pull/4484)
- [x] Multiple UI and UX enhancements to Timeline (https://github.com/elastic/security-docs/pull/4299, https://github.com/elastic/security-docs/pull/4298, https://github.com/elastic/security-docs/pull/4483)
[Timeline](https://www.elastic.co/guide/en/security/master/timelines-ui.html) now opens as a modal, has an option to manually save changes, and has undergone significant UI changes. For example, the [query builde](https://www.elastic.co/guide/en/security/master/timelines-ui.html#narrow-expand)r is now collapsible, which allows you to have more space for Timeline results.
- [x] Feature flag added for the **ES|QL** tab (https://github.com/elastic/security-docs/pull/4552).
You can now remove the ES|QL tab by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
- [x] Default ES|QL query removed (https://github.com/elastic/security-docs/pull/4578).
The default query was removed for increased tab performance. (I'm going to follow up with Michael on this next Tuesday to see if he wants this doc'd in What's new and for specific language.)

## Generative AI

- [x] Elastic AI Assistant now supports [retrieval-augmented generation (RAG) for alerts](https://github.com/elastic/security-docs/pull/4520). Using this feature, you can provide information about multiple alerts to AI Assistant so that it can answer a broader scope of questions relating to alerts in your environment. 

## EDR Workflows/Asset Management 

- [x] Bidirectional integration response actions (SentinelOne) (https://github.com/elastic/security-docs/pull/4593).
Powered by the [SentinelOne integration](https://docs.elastic.co/integrations/sentinel_one) for Elastic Agent, SentinelOne response actions allow you to perform bidirectional actions on protected hosts, such as directing SentinelOne to isolate a suspicious endpoint from your network, without needing to leave the Elastic Security UI. 

- [x] Event filters and endpoint exceptions support for `matches` and `does not match` conditions (https://github.com/elastic/security-docs/pull/4602).
You can now use `matches` and `does not match` conditions on more fields when configuring event filters and endpoint exceptions. Previously, only the `file.path.text` field was supported.

- [x] New Osquery query timeout setting (https://github.com/elastic/security-docs/pull/4611, https://github.com/elastic/kibana/pull/174595).
Allows you to set timeout period, in seconds, after which the query will stop running. Overwriting the default timeout allows you to support queries that require more time to complete. The default and minimum supported value is `60`. The maximum supported value is `900`.

## Cloud Security 

- [x] Cloud Security Posture Management [can now be deployed](https://github.com/elastic/security-docs/pull/4551) to entire Microsoft Azure organizations.
- [x] The [Findings page](https://github.com/elastic/security-docs/pull/4532) now enables you to group your data by any field, and to further customize how it is displayed.

## Endpoint 

- [ ] Add features here. 

## Protections Experience

- [ ] Add features here. 

## ResponseOps

- [ ] Believe these will be doc'd in Kibana's "What's new for 8.12" page.

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves #4465. 

Preview: [What’s new in 8.12](https://security-docs_bk_4612.docs-preview.app.elstc.co/guide/en/security/master/whats-new.html)

### CODE REVIEW

File: docs/whats-new.asciidoc [hunk 1]
```diff
@@ -4,105 +4,106 @@
...
 {security-guide-all}/7.9/whats-new.html[7.9]
 
 // NOTE: The notable-highlights tagged regions are re-used in the Installation and Upgrade Guide. Full URL links are required in tagged regions.
 // tag::notable-highlights[]
 
+[float]
+== Retrieval-augmented generation supported in Elastic AI Assistant
```
Comment (benironside) [inline docs/whats-new.asciidoc:14]:
```suggestion
== Retrieval-augmented generation for alerts in Elastic AI Assistant
```
There's already RAG for ES|QL so I think this additional specificity is warranted

File: docs/whats-new.asciidoc [hunk 2]
```diff
@@ -4,105 +4,106 @@
...
-With the new risk scoring engine, you can:
+[float]
+=== JSON diff for Elastic prebuilt rule updates
 
-* Preview and enable the risk engine using a centralized one-click onboarding workflow.
-* Conveniently migrate to the new engine if you're an existing user of risk scoring.
-* Generate risk scores for hosts and users from the last 30 days.
-* View the alerts that contributed to an entity's risk score, allowing faster investigations.
-* Continue to access entity risk analytics in existing security workflows.
+When Elastic updates a prebuilt detection rule, you can examine the latest version before you {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[update] to it. The rule details flyout in **Rule Updates** displays a side-by-side JSON comparison of the rule's **Base version** (what you currently have installed) and the **Update version** that you can choose to install.
```
Comment (benironside) [inline docs/whats-new.asciidoc:26]:
```suggestion
When Elastic updates a prebuilt detection rule, you can examine the latest version before you {security-guide}/prebuilt-rules-management.html#update-prebuilt-rules[update] it. The rule details flyout in **Rule Updates** displays a side-by-side JSON comparison of the rule's **Base version** (what you currently have installed) and the **Update version** that you can choose to install.
```

**Review by benironside — COMMENTED** [review]
LGTM, just left two minor suggestions

File: docs/whats-new.asciidoc [hunk 3]
```diff
@@ -4,105 +4,106 @@
...
 
-The following enhancements have been added to detection rules and alerts:
+The following enhancements have been added to Timelines:
 
 [float]
-=== Create ES|QL query detection rules with new ES|QL rule type
+=== Multiple UI and UX enhancements to Timeline
 
-Use the new {security-guide}/rules-ui-create.html#create-esql-rule[ES|QL rule type] to create detection rules that use ES|QL queries. The ES|QL rule type supports aggregating and non-aggregating queries.
+{security-guide}/timelines-ui.html[Timeline] now opens as a modal, requires you to manually save changes, and has an option to save changes as a new Timeline. In addition, Timeline has undergone significant UI changes. For example, the query builder is now collapsible, which allows you to have more space for Timeline results.
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:52]:
```suggestion
{security-guide}/timelines-ui.html[Timeline] now opens as a modal, requires you to manually save changes, and has the option to save changes as a new Timeline. Additional UX improvements have been also introduced. For example, the query builder is now collapsible, which allows you to have more space for Timeline results.
```

File: docs/whats-new.asciidoc [hunk 4]
```diff
@@ -4,105 +4,106 @@
...
 [role="screenshot"]
-image::whats-new/images/8.11/esql-rule.png[New ES|QL rule type]
+image::whats-new/images/8.12/timeline-ui-updated.png[Updated Timeline UI]
 
 [float]
-=== Case-sensitive values supported in rule exceptions
+=== Feature flag added for the {esql} tab
 
-When {security-guide}/add-exceptions.html#detection-rule-exceptions[adding exceptions to a rule], the `is one of` and `is not one of` operators now support identical, case-sensitive values – for example, `Windows` and `windows`.
+You can now remove the {security-guide}/timelines-ui.html#esql-in-timeline[{esql} tab] by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:60]:
```suggestion
You can now remove the {security-guide}/timelines-ui.html#esql-in-timeline[**{esql} tab**] by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
```

File: docs/whats-new.asciidoc [hunk 5]
```diff
@@ -4,105 +4,106 @@
...
 
 [float]
-== Use ES|QL in Timeline
+=== Default {esql} query removed from the {esql} tab
 
-You can use {security-guide}/timelines-ui.html#esql-in-timeline[ES|QL in Timeline] to filter, transform, and analyze event data stored in {es}. To start using ES|QL, open the **ES|QL** tab.
+The default {esql} query was removed from the {esql} tab, for increased tab performance.
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:65]:
```suggestion
The default {esql} query was removed from the **{esql}** tab, for increased tab performance.
```

File: docs/whats-new.asciidoc [hunk 6]
```diff
@@ -4,105 +4,106 @@
...
+
+[float]
+== Exclude cold and frozen tiers from analyzer queries
+
+You can now exclude cold and frozen tier data from visual event analyzer queries, to increase analyzer performance. You can do this by turning on the `securitySolution:excludeColdAndFrozenTiersInAnalyzer` {security-guide}/advanced-settings.html#exclude-cold-frozen-tiers[advanced setting].
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:70]:
```suggestion
You can now exclude cold and frozen tier data from visual event analyzer queries to increase analyzer performance. You can do this by turning on the `securitySolution:excludeColdAndFrozenTiersInAnalyzer` {security-guide}/advanced-settings.html#exclude-cold-frozen-tiers[advanced setting].
```

**Review by nastasha-solomon — APPROVED** [review]
Looks great - thank you!

File: docs/whats-new.asciidoc [hunk 7]
```diff
@@ -4,105 +4,106 @@
...
 [role="screenshot"]
-image::whats-new/images/8.11/esql-rule.png[New ES|QL rule type]
+image::whats-new/images/8.12/timeline-ui-updated.png[Updated Timeline UI]
 
 [float]
-=== Case-sensitive values supported in rule exceptions
+=== Feature flag added for the {esql} tab
 
-When {security-guide}/add-exceptions.html#detection-rule-exceptions[adding exceptions to a rule], the `is one of` and `is not one of` operators now support identical, case-sensitive values – for example, `Windows` and `windows`.
+You can now remove the {security-guide}/timelines-ui.html#esql-in-timeline[**{esql} tab**] by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:60]: Sorry, just one more tiny correction (to correct my earlier correction) here:

```suggestion
You can now remove the {security-guide}/timelines-ui.html#esql-in-timeline[**{esql}**] tab by editing your {cloud}/ec-manage-kibana-settings.html#ec-manage-kibana-settings[{kib} user settings] and adding the `xpack.securitySolution.enableExperimental: ["timelineEsqlTabDisabled"]` feature flag.
```

**Review by joepeeples — APPROVED** [review]

**Pull request merged**